docs/api/rest/config.md

```diff
@@ -156,19 +96,19 @@ curl --location --request POST 'http://localhost:5000/api/config/ClaraAeTitle/'
 
 ---
 
-## DELETE /api/config/sourceaetitle/{ae-title}
+## DELETE /api/config/claraaetitle/{ae-title}
 
-Deletes a Source AE Title.
+Deletes a Clara AE Title.
```

Comment:
May want to call out Clara SCP AE Title, that is what I think is.

Reply:
Fixed